app-frontend/src/app/pages/settings/profile/profile.html

```diff
@@ -67,7 +68,7 @@ <h1 class="h3 page-title">Profile</h1>
           <p class="help-block">Contact support to change your username</p>
         </div>
         <button type="button"
-                ng-if="$ctrl.provider === 'auth0'"
+                ng-if="$ctrl.provider.provider === 'auth0'"
```

Comment:
is this ng-if actually necessary? It's already inside another nv-if that tests the same thing right?